app/services/foreman/plugin.rb

```diff
@@ -93,6 +93,17 @@ def installed?(id)
 
     def initialize(id)
       @id = id.to_sym
+      add_loggers
+    end
+
+    def add_loggers
+      return unless (defined?(Foreman::Logging) && SETTINGS[@id])
+
+      if (loggers = SETTINGS[@id][:loggers])
```

Comment:
AFAIK this would be the same condition as if a logger is disabled (i.e. nothing would get output to the log file).